docker-compose.yaml

```diff
@@ -1,40 +1,61 @@
 version: '3.8'
+
+networks:
+  openfga:
+
 services:
-  postgres:
+  db:
     image: postgres:14
-    container_name: postgres
     networks:
-      - default
-    ports:
-      - "5432:5432"
+      - openfga
     environment:
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=password
+      - POSTGRES_DB=openfga
```

Comment:
If this is changed, this postgres container will no longer work with `make run-postgres`. I think it is nice to be able to `docker compose up -d postgres` and then `make run-postgres`.

Reply:
I'm not sure what that means. Was the default db name of `postgres` used in the past?